test/parallel/test-timer-immediate.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 const common = require('../common');
 common.globalCheck = false;
+// eslint-disable-next-line no-global-assign
 process = {};  // Boom!
```

Comment:
I think you can make it not need the comment by doing `global.process = {}` instead?